src/routes/generator/+page.svelte

```diff
@@ -1,20 +1,26 @@
 <script lang="ts">
-	import { isMapModalOpen, files } from './stores';
+	import { isMapModalOpen, files, startModalOpen, imageLinkFromUser } from './stores';
 	import MapModal from './MapModal.svelte';
 	import Toolbar from './Toolbar.svelte';
 	import Canvas from './Canvas.svelte';
-	import PlaceHolder from './Placeholder.svelte';
+	import { onMount } from 'svelte';
+	import StartModal from './StartModal.svelte';
+
+	onMount(() => {
+		$startModalOpen = true;
+	});
 </script>
 
+{#if $startModalOpen && ($files == null || $imageLinkFromUser == null)}
+	<StartModal />
+{/if}
+
 {#if $isMapModalOpen}
 	<MapModal />
 {/if}
 
 <div class="pageWrapper">
 	<Toolbar />
-	{#if !$files}
-		<PlaceHolder />
```

Comment:
If you are not using the placeholder anymore, should the file be deleted too?

Reply:
removed file.